api/helpers/find-or-create-current-match.js

```diff
@@ -0,0 +1,70 @@
+const dayjs = require('dayjs');
+
+module.exports = {
+  friendlyName: 'Find Or Create Current Match Between Two Players',
+
+  description:
+    'Finds the relevant match for the specified players for the current week, and creates the match if it does not already exist',
+
+  inputs: {
+    player1: {
+      type: 'number',
+      descritiption: 'The database id of the first player in the desired match',
+      example: 44,
+      required: true,
+    },
+    player2: {
+      type: 'number',
+      descritiption: 'The database id of the second player in the desired match',
+      example: 45,
+      required: true,
+    },
+  },
+
+  fn: async ({ player1, player2 }, exits) => {
```

Comment:
This helper answers one of my above questions, haha. Should it fast fail if both players don't exist?

Reply:
Like look up if users with those ids don't exist? An issue with that is that currently we don't really need to look up the users themselves, so it adds an extra database query for each player in order to fail fast, which feels not fast by comparison. I'm open to it, but we'd likely want to rework the query then to avoid the use of `.populate()` in order to prevent redundant queries